sdk/eventhub/Azure.Messaging.EventHubs/src/Producer/EventHubProducerClient.cs

```diff
@@ -370,53 +370,64 @@ public virtual async Task<string[]> GetPartitionIdsAsync(CancellationToken cance
+        ///   Occurs when the set of events exceeds the maximum size allowed in a single batch, as determined by the Event Hubs service.  The <see cref="EventHubsException.Reason" /> will be set to
+        ///   <see cref="EventHubsException.FailureReason.MessageSizeExceeded"/> in this case.
+        /// </exception>
+        ///
+        /// <seealso cref="SendAsync(IEnumerable{EventData}, SendEventOptions, CancellationToken)" />
+        /// <seealso cref="SendAsync(EventDataBatch, CancellationToken)" />
+        /// <seealso cref="CreateBatchAsync(CancellationToken)" />
         ///
-        internal virtual async Task SendAsync(IEnumerable<EventData> events,
-                                              CancellationToken cancellationToken = default) => await SendAsync(events, null, cancellationToken).ConfigureAwait(false);
+        public virtual async Task SendAsync(IEnumerable<EventData> eventBatch,
+                                            CancellationToken cancellationToken = default) => await SendAsync(eventBatch, null, cancellationToken).ConfigureAwait(false);
 
         /// <summary>
-        ///   Sends a set of events to the associated Event Hub using a batched approach.  If the size of events exceed the
-        ///   maximum size of a single batch, an exception will be triggered and the send will fail.
+        ///   Sends a set of events to the associated Event Hub using a batched approach.  Unlike the <see cref="SendAsync(EventDataBatch, CancellationToken)" /> overload,
+        ///   this method may fail if the size of events exceed the maximum allowable size of a single batch.
         /// </summary>
         ///
-        /// <param name="events">The set of event data to send.</param>
+        /// <param name="eventBatch">The set of event data to send.</param>
         /// <param name="options">The set of options to consider when sending this batch.</param>
         /// <param name="cancellationToken">An optional <see cref="CancellationToken" /> instance to signal the request to cancel the operation.</param>
         ///
         /// <returns>A task to be resolved on when the operation has completed.</returns>
         ///
-        /// <seealso cref="SendAsync(EventData, CancellationToken)" />
-        /// <seealso cref="SendAsync(EventData, SendEventOptions, CancellationToken)" />
+        /// <exception cref="EventHubsException">
+        ///   Occurs when the set of events exceeds the maximum size allowed in a single batch, as determined by the Event Hubs service.  The <see cref="EventHubsException.Reason" /> will be set to
+        ///   <see cref="EventHubsException.FailureReason.MessageSizeExceeded"/> in this case.
+        /// </exception>
+        ///
         /// <seealso cref="SendAsync(IEnumerable{EventData}, CancellationToken)" />
         /// <seealso cref="SendAsync(EventDataBatch, CancellationToken)" />
+        /// <seealso cref="CreateBatchAsync(CancellationToken)" />
```

Comment:
It may be better to link this to `CreateBatchAsync(CreateBatchOptions, CancellationToken)` since this `SendAsync` overload takes `SendEventOptions`.
```suggestion
        /// <seealso cref="CreateBatchAsync(CreateBatchOptions, CancellationToken)" />
```

Reply:
Good call.   Applying this directly, as committing a suggestion tends to really upset my local rebasing.